# HG changeset patch
# User Rik <rik@octave.org>
# Date 1468858139 25200
#      Mon Jul 18 09:08:59 2016 -0700
# Node ID 3459c25b679a6b776d0f9a18bbecdd574e281b56
# Parent  37f41acc2c39447370880c45f573cb59f8e102e6
__unimplemented__.m: Remove condeig from list.

* __unimplemented__.m: Remove condeig from list.

diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -600,17 +600,16 @@ function list = missing_functions ()
   "checkout",
   "clearvars",
   "clipboard",
   "cmopts",
   "colordef",
   "colormapeditor",
   "commandhistory",
   "commandwindow",
-  "condeig",
   "coneplot",
   "containers.Map",
   "contourslice",
   "corrcoef",
   "countcats",
   "createClassFromWsdl",
   "createSoapMessage",
   "customverctrl",
